packages/unix_home.rb

```diff
@@ -0,0 +1,48 @@
+require 'package'
+
+class Unix_home < Package
```

Comment:
Is the idea here to make sure the home directory has execute permissions?

Reply:
Yes, and get rid of the `nosymfollow` mount symbol